app/services/transaction_service/transaction.rb

```diff
@@ -48,19 +48,15 @@ def can_start_transaction_braintree(community_id:, author_id:)
   end
 
   def can_start_transaction_paypal(community_id:, author_id:)
-    personal_account_verified = paypal_account_verified?(community_id: community_id, person_id: author_id)
+    payment_settings = Maybe(PaymentSettingsStore.get_active(community_id: community_id))
+                       .select {|set| set[:payment_gateway] == :paypal && set[:commission_from_seller] && set[:minimum_price_cents]}
```

Comment:
The contents of select-block could be extracted to method. Would be faster to read and better documented.